lib/mayaUsd/fileio/translators/translatorMesh.cpp

```diff
@@ -298,11 +299,20 @@ TranslatorMeshRead::TranslatorMeshRead(
     // Animate the weights so that mesh0 has a weight of 1 at frame 0, etc.
     MFnAnimCurve animFn;
 
+    // Get the values needed to convert time to the current maya scenes framerate
+    MTime::Unit timeUnit = MTime::uiUnit();
+    if (!context) {
+        TF_RUNTIME_ERROR("Received null context when writing <%s>", prim.GetPath().GetText());
+        *status = MS::kFailure;
+        return;
+    }
+    double timeSampleMultiplier = context->GetTimeSampleMultiplier();
```

Comment:
The original intent with these translator classes was that they could be used outside the context of an import operation to do "in-memory" translations, so you weren't necessarily required to have a context. I'm not sure whether that's still workable now, but I think we'd eventually want to provide that support, so I'd suggest allowing `context` to be `nullptr`:

Allow context to be `nullptr`:
```C++
MTime::Unit timeUnit = MTime::uiUnit();
double      timeSampleMultiplier = (context != nullptr) ? context->GetTimeSampleMultiplier() : 1.0;
```